src/main/zapHomeFiles/hudtutorial/en_GB/SiteAlerts.html

```diff
@@ -15,8 +15,8 @@ <h1>Site Alerts</H1>
 	The alerts are grouped by risk in the same way as Page Alerts:
 	<ul>
 	<li><img src="site-alerts-high.png"> High Level Site Alerts
-	<li><img src="site-alerts-low.png"> Low  Level Site Alerts
 	<li><img src="site-alerts-medium.png"> Medium Level Site Alerts
+	<li><img src="site-alerts-low.png"> Low Level Site Alerts
```

Comment:
Should we remove all occurrences of "Site" here as with the Page list?

In both cases should the `li` have closing tags?

Reply:
For consistency, if we don't mind the differences in the labels.

Fine either way.